server/routes/index.js

```diff
@@ -7,7 +7,13 @@ var renderAdminPage = require('../renderAdminPage');
 router.route('/').all(auth);
```

Comment:
I think this will cause the homepage to be only accessible to users who can log-in through S30. I think we should move the auth middleware declaration to after the home page route on lines 10-16, that way users can become viewers without logging in and vanity redirects will still require users to log-in.